Models/Chat.cs

```diff
@@ -1,21 +1,28 @@
 ﻿using System;
 using System.Collections.Generic;
 
-namespace Tinker_Back.Models;
-
-public partial class Chat
+namespace Tinker_Back.Models
 {
-    public int Id { get; set; }
 
-    public string Name { get; set; } = null!;
+    public class Chat
+    {
+        public Chat(string name, string description)
+        {
+            Name = name;
+            Description = description;
+        }
+        public int Id { get; set; }
+
+        public string Name { get; set; } = null!;
 
-    public bool? IsGroupChat { get; set; }
+        public bool? IsGroupChat { get; set; }
```

Comment:
Do we need nullable option in booleans, probably it will be better to set their default values in constructor
About groupChat it can be done, by passing 1 more parameter to constructor